pkg/cluster/failurediagnostics/virtualmachines.go

```diff
@@ -0,0 +1,86 @@
+package failurediagnostics
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"bufio"
+	"context"
+	"encoding/base64"
+	"fmt"
+
+	mgmtstorage "github.com/Azure/azure-sdk-for-go/services/storage/mgmt/2019-06-01/storage"
+
+	"github.com/Azure/ARO-RP/pkg/util/stringutils"
+)
+
+func (m *manager) LogAzureInformation(ctx context.Context) (interface{}, error) {
+	items := make([]interface{}, 0)
+
+	if m.virtualMachines == nil {
+		items = append(items, "vmclient missing")
+		return items, nil
+	}
+
+	resourceGroupName := stringutils.LastTokenByte(m.doc.OpenShiftCluster.Properties.ClusterProfile.ResourceGroupID, '/')
+	vms, err := m.virtualMachines.List(ctx, resourceGroupName)
+	if err != nil {
+		items = append(items, fmt.Sprintf("vm listing error: %s", err))
+		return items, nil
+	}
+
+	if len(vms) == 0 {
+		items = append(items, "no vms found")
+		return items, nil
+	}
+
+	consoleURIs := make([][]string, 0)
```

Comment:
I think `consoleURI` is confusable with `consoleURL`.
I'd suggest `BootDiagURIs` or `SerialConsoleLogBlobURIs` (too long?)